src/components/Contentful/SectionWithBackgroundClassic.vue

```diff
@@ -150,8 +150,10 @@ export default {
 				if (key === 'sm') hasSmSizes = true;
 				if (key === 'lg') hasLgSizes = true;
 				// process sizes
-				const top = sizeObj.top.toString() ? `${sizePrefix}tw-pt-${sizeObj.top}` : '';
-				const bottom = sizeObj.bottom.toString() ? `${sizePrefix}tw-pb-${sizeObj.bottom}` : '';
```

Comment:
I think this can be simplified with optional chaining to
```const bottom = sizeObj.bottom?.toString() ? `${sizePrefix}tw-pb-${sizeObj.bottom}` : '';```